.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+# All jobs have the same outcome. We define multiple for resiliency reasons.
+jobs:
+  # In thechangelog/pipedream repository (a.k.a. upstream),
+  # this is the preferred default:
+  on-namespace:
+    if: ${{ contains(vars.RUNS_ON, 'namespace') }}
+    uses: ./.github/workflows/_namespace.yml
+    secrets: inherit
+
+  # Just in case Namespace.so misbehaves, we want a fallback:
```

Comment:
Is `.so` intentional? I don't see it anywhere but in comments here.

Reply:
Yes.

Searching for `Namespace` returns a bunch of unrelated results, including "K8s namespaces", "Namespace.com" & programming references (C++, C#, TypeScript, etc.). The `.so` is meant to disambiguates it.

This is a good point though. I will capture it in a few more places so that it's clearer.